src/js/components/search/visualizations/time/TimeVisualizationSection.jsx

```diff
@@ -70,7 +70,7 @@ export default class TimeVisualizationSection extends React.Component {
         <>
             <div className="tooltip__title">Download data by {capitalize(this.props.data.visualizationPeriod === 'fiscal_year' ? 'year' : this.props.data.visualizationPeriod)}</div>
             <div className="tooltip__text">
-                Download a CSV of award spending data that matches your search criteria, broken down by year. For complete download results, click on the &quot;Download&quot; button on the top right of this page.
+                Download a CSV of award spending data that matches your search criteria, broken down by {this.props.data.visualizationPeriod === 'fiscal_year' ? 'year' : this.props.data.visualizationPeriod}. For complete download results, click on the &quot;Download&quot; button on the top right of this page.
```

Comment:
I do not believe this change should be here. @brett-varney created a warm-fix into QAT for a change relevant to this and was down merged.

Reply:
True

Reply:
It shouldn't be HERE, but should already be in dev, so I don't know why it thinks this is a change...